### PR TITLE
Pytest refactor

### PR DESCRIPTION
**Pull request by jonparrott**
@waprin this is the culmination of all the reviews you've been doing. This is the final merge into master.

### CODE REVIEW

File: 4-auth/bookshelf/model_cloudsql.py [hunk 1]
```diff
@@ -85,7 +83,9 @@ def read(id):
 
 
 def create(data):
+    print(data)
```
Comment (waprin) [inline 4-auth/bookshelf/model_cloudsql.py]: intentional debug statements or accidental commit?

**Comment by waprin** [comment]
lgtm if you want to merge, but I need some help getting the tests working locally. I think we should at least create an issue to document running the tests better.

File: 4-auth/bookshelf/model_cloudsql.py (hunk 1 above)
Comment (jonparrott) [inline 4-auth/bookshelf/model_cloudsql.py]: Yeah, I'll fix.

**Comment by jonparrott** [comment]
@waprin definitely want to make sure you can run locally first. We can chat after Next on getting that setup.

**Pull request merged**